### PR TITLE
Distinguish between template availability and support status

### PR DESCRIPTION
**Pull request by andrewdavidwong**
QubesOS/qubes-issues#5489

### CODE REVIEW

File: user/downloading-installing-upgrading/supported-versions.md [hunk 1]
```diff
@@ -75,6 +73,45 @@ Currently, only [Fedora] and [Debian] TemplateVMs are officially supported by th
...
+
+
+### Debian
+
+Qubes support for each [Debian] TemplateVM ends when that Debian release reaches [end-of-life][debian-eol], unless otherwise noted.
+
+| TemplateVM    | Status                  |
+| ------------- | ----------------------- |
+| Debian 7      | Old, unsupported        |
+| Debian 8      | Old, unsupported        |
```
Comment (andrewdavidwong) [inline user/downloading-installing-upgrading/supported-versions.md]: @marmarek, IIRC, you started a discussion about dropping support for Debian 8 on qubes-devel. I don't recall seeing a resolution to that question. Did you decide to drop support? If so, how should we represent that here?

Comment (marmarek) [reply user/downloading-installing-upgrading/supported-versions.md]: > Did you decide to drop support?

Yes.

> If so, how should we represent that here?

Something like this. TBH I don't understand what "old"/"current" means in this table - is it just another word for "unsupported"/"supported"?

**Comment by marmarek** [comment]
I think there needs to be also information about Qubes release. For example currently Qubes Fedora 31 packages are available ("supported", if that applies in this case) for R4.1, but not R4.0.

File: user/downloading-installing-upgrading/supported-versions.md (hunk 1 above)
Comment (andrewdavidwong) [reply user/downloading-installing-upgrading/supported-versions.md]: >  TBH I don't understand what "old"/"current" means in this table - is it just another word for "unsupported"/"supported"?

"Old" means that it is not the newest release. An old release can still be supported. For example, when Qubes 4.0 came out, 3.2 was still supported, but it was the old version.

However, we can drop "old"/"new" in order to simplify the table.

**Comment by andrewdavidwong** [comment]
> I think there needs to be also information about Qubes release. For example currently Qubes Fedora 31 packages are available ("supported", if that applies in this case) for R4.1, but not R4.0.

That's what the existing table is for. It already reflects this kind of information.

However, it has not yet been updated with info about Fedora 31 and Qubes 4.1. I can update that now, as a separate matter from this PR.

**Comment by andrewdavidwong** [comment]
Done.

**Pull request merged**